roles/replace_node/tasks/peers.yml

```diff
@@ -7,6 +7,10 @@
   register: old_node_uuid
   run_once: true
 
+- name: Fail if parsed hostname is different from the back-end FQD
+  fail: msg="Hostname parsed in the playbook differs from the back-end gluster FQDN"
```

Comment:
In msg please remove parse, you can mention like: "Hostname mentioned in inventory should be same with back-end gluster FQDN"